doc/crypto/api/ops/signature.rst

```diff
@@ -463,20 +473,56 @@ EdDSA signature algorithms
 
```

Comment:
Three lines up (470): The algorithm can also be used with the sign/verify message with context functions. At least for Edwards448 curve keys.